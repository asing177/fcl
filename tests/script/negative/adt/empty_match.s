type t { C }

@initial
run(t arg) {
  case(arg) { }
}
