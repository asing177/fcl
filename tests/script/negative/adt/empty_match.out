run(t arg) {
  case(arg) { }
--------------^
}

"<stdin>" (line 5, column 15):
unexpected "}"
expecting literal, name, uppercase letter or "_"
TimeDelta must not be 0